libs/application/templates/data-protection-complaint/src/lib/dataSchema.ts

```diff
@@ -0,0 +1,63 @@
+import * as z from 'zod'
+import { YES, NO } from '../shared'
+import { error } from './messages/error'
+
+export enum OnBehalf {
+  MYSELF = 'myself',
+  MYSELF_AND_OR_OTHERS = 'myselfAndOrOthers',
+  COMPANY = 'company',
+  ORGANIZATION_OR_INSTITUTION = 'organizationOrInsititution',
+}
+
+const nationalIdRegex = /([0-9]){6}-?([0-9]){4}/
```

Comment:
use the kennitala package (this regex isn't correct for national Ids):
`import * as kennitala from 'kennitala'`

Reply:
e.g. `1234567890` would pass, but isn't valid.

Reply:
Sounds good, we might wanna inform the other teams about this too